.github/CONTRIBUTING.md

```diff
@@ -12,3 +12,75 @@ That documentation can be found:
 2. Write tests if you add any new functionality. 
 2. Increase the version numbers in the `package.json` to the new version that this Pull Request would represent. The versioning scheme we use is [SemVer](https://semver.org/).
 3. You may merge the Pull Request in once you have the sign-off of at least one other developer, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+
+## Release a new version
+### Prerequisites
```

Comment:
Headings need to have a blank line before and after (except if a heading is the first line within a markdown file).